types/flatbuffers/index.d.ts

```diff
@@ -167,7 +169,8 @@ declare global {
        */
       endObject(): Offset;
 
-      finish(root_table: Offset, file_identifier?: string): void;
+      finish(root_table: Offset, file_identifier?: string, size_prefix?: boolean): void;
+      finishSizePrefixed(root_table: Offset, file_identifier?: string): void;
```

Comment:
same here, that's new, would be nice to have the method added to tests as well.

Reply:
Done. Keep in mind these tests don't appear to be _real_ tests, they're just making sure it all compiles.